### ISSUE TITLE
Hulk theme listed in Themes.md does not exist in repos

### ISSUE THREAD

**Issue by NickBrisebois** [oh-my-fish/oh-my-fish#800]
The Hulk theme listed [here](https://github.com/oh-my-fish/oh-my-fish/blob/master/docs/Themes.md#hulk) doesn't seem to exist. I've looked for the repository itself on the oh-my-fish organization but can't seem to find it. I really like the theme, does it exist anywhere or should it be removed from Themes.md?

![image](https://user-images.githubusercontent.com/3811648/104648795-abeae500-5681-11eb-9865-d8341f7aea29.png)

**Environment report**

```
Oh My Fish version:   7
OS type:              Linux
Fish version:         fish, version 3.1.0
Git version:          git version 2.25.1
Git core.autocrlf:    input
Checking for a sane environment...
Your shell is ready to swim.
```

**Comment by scorphus** [comment]
Very unfortunately, the hulk theme was removed by the author and we have no control over that.

**Comment by scorphus** [comment]
Reopening this as we haven't yet merged #806.

**Comment by scorphus** [comment]
Thanks for reporting, @NickBrisebois
